docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -228,6 +230,12 @@ The (optional) name of the Podman container. If this is not specified, the defau
 of `systemd-%N` is used, which is the same as the service name but with a `systemd-`
 prefix to avoid conflicts with user-managed containers.
 
+### `ContainersConfModule=`
```

Comment:
Just a small nit but maybe rename to `GlobalArgs=`? That would bring in some consistency with `PodmanArgs=` which is similar.

Reply:
Since both you and @vrothberg  suggested `GlobalArgs`, I went with that.